trade-finance-manager-ui/server/middleware/validateSqlId/index.ts

```diff
@@ -0,0 +1,13 @@
+import { NextFunction, Request, Response } from 'express';
+import { isValidSqlId } from '../../helpers/validateIds';
+
+export const validateSqlId = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
can you add a quick docstring at the top:
`/**
Middleware to validate that the 'id' field found as a path param is a valid SQL id
*/`